### PR TITLE
Make global repl explicit + docs improvements

### PR DESCRIPTION
**Pull request by allentiak**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -49,11 +56,7 @@ In addition to the REPL development inside a single module, it's possible to run
 
 ## Style Guide
 
-The Clojure code in this project follows the [Clojure Style Guide][2].
-
-### Exceptions:
-
-* we use two empty lines between top-level forms
+The Clojure code in this project mainly follows the [Clojure Style Guide][2], except that we use 2 (two) empty lines between top-level forms.
```
Comment (alexanderkiel) [inline DEVELOPMENT.md:59]: We no longer use two empty lines between functions. In fact we use `cljfmt` now. So it would be good to mention `cljfmt` here.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
Thanks for your contribution. Can you please rebase on develop and change the merge target to develop also?

File: DEVELOPMENT.md (hunk 1 above)
Comment (allentiak) [reply DEVELOPMENT.md:59]: What about this?

"The Clojure code in this project mainly follows the [Clojure Style Guide][2], enforced by `cljfmt`. For more details, please check `cljfmt.edn`."

Comment (alexanderkiel) [reply DEVELOPMENT.md]: Perfect

**Comment by allentiak** [comment]
It seems there is a token for running all the codecov tests that is missing...

I think there is an automated PR that fixes that...
https://github.com/samply/blaze/pull/1687

**Pull request merged**